### PR TITLE
Add more folders

### PR DESCRIPTION
**Pull request by HisMaj3sty**

### CODE REVIEW

File: frontend/src/lib/Session/SessionFeed.svelte [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <script lang="ts">
-  import Sessionname from '$lib/SessionName.svelte';
-  import StudentsList from '$lib/StudentsList.svelte';
+  import Sessionname from './SessionName.svelte';
+  import StudentsList from './StudentsList.svelte';
   import OutlineButton from '$lib/OutlineButton.svelte';
-  import type { Student } from '$lib/student.js';
+  import type { Student } from '$lib/API/student.js';
```
Comment (DCNick3) [inline frontend/src/lib/Session/SessionFeed.svelte:5]: please no .js, remove the extension (for consistency) (even though it works bc of some remapping)

it seems that imports w/o the extension work too and seem to be used the most in our code

File: frontend/src/routes/sessions/+page.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { Session } from '$lib/session';
-import { load_with_api } from '$lib/api';
+import { Session } from '$lib/QRcode/session.js';
```
Comment (DCNick3) [inline frontend/src/routes/sessions/+page.ts]: ditto s/js//

**Review by DCNick3 — CHANGES_REQUESTED** [review]

File: frontend/src/lib/Session/SessionFeed.svelte (hunk 1 above)
Comment (HisMaj3sty) [reply frontend/src/lib/Session/SessionFeed.svelte]: Fixed

File: frontend/src/routes/sessions/+page.ts (hunk 2 above)
Comment (HisMaj3sty) [reply frontend/src/routes/sessions/+page.ts]: Fixed

**Review by DCNick3 — APPROVED** [review]

**Pull request merged**